pom.xml

```diff
@@ -32,7 +32,7 @@
         <dependency>
             <groupId>xyz.grasscutters</groupId> <!-- Versions below 1.0.3-dev are on 'tech.xigam' -->
             <artifactId>grasscutter</artifactId>
-            <version>1.2.2-dev</version> <!-- Replace with the latest version of the API. -->
```

Comment:
I know that 1.2.3-dev is the newest version but it changes annotations (I might be wrong tho)
![image](https://user-images.githubusercontent.com/81880849/181044151-ae79822b-45ad-4a06-9c8e-4ab4cb7a3944.png)
But from what I and hartie95#8966 tested 1.2.2-dev works also on 1.2.3-dev